modules/500-upmeter/templates/smoke-mini/statefulsets.yaml

```diff
@@ -73,7 +73,7 @@ spec:
                 operator: In
                 values:
                 - {{ $sts.node | quote }}
-      {{- include "helm_lib_tolerations" (tuple $context "any-node") | nindent 6 }}
+      {{- include "helm_lib_tolerations" (tuple $context "any-node" "without-storage-problems") | nindent 6 }}
```

Comment:
```suggestion
{{- if ne $storageClass "false" }}
      {{- include "helm_lib_tolerations" (tuple $context "without-storage-problems" "without-storage-problems") | nindent 6 }}
{{- else}}
      {{- include "helm_lib_tolerations" (tuple $context "any-node" "without-storage-problems") | nindent 6 }}
{{- end }}
```

Reply:
In general, smoke-mini should always schedule on any node regardless of storage problems. synthetic/disk probe is disabled by default, so it's up to user to use the probe along with storage problems.

I suggest keeping "any-node".